analyses/survival-analysis/run_survival.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
 
-# K S Gaonkar
+# K S Gaonkar, J Rokita
 
-# Run fusion_filtering
+# Run survival-analysis
 
 set -e
 set -o pipefail
 
```

Comment:
Adding this will allow the script to run with paths relative to its location (this directory).
```suggestion
# Set the working directory to the directory of this file
cd "$(dirname "${BASH_SOURCE[0]}")"
```